controllers/activemqartemisaddress_controller_test.go

```diff
@@ -1101,3 +1176,64 @@ func CheckQueueExistInPod(brokerCrName string, podName string, queueName string,
 		g.Expect(content).Should(ContainSubstring(queueName))
 	}, existingClusterTimeout, existingClusterInterval).Should(Succeed())
 }
+
+func CheckQueueAttribute(brokerCrName string, podName string, namespace string, queueName string, addressName string, routingType string, attrName string, attrValueAsString string) {
+
+	if os.Getenv("USE_EXISTING_CLUSTER") != "true" {
+		Fail("function should be called with an existing cluster")
+	}
+
+	Eventually(func(g Gomega) {
+		key := types.NamespacedName{Name: namer.CrToSS(brokerCrName), Namespace: namespace}
+		sfsFound := &appsv1.StatefulSet{}
+
+		g.Expect(k8sClient.Get(ctx, key, sfsFound)).Should(Succeed())
+		g.Expect(sfsFound.Status.ReadyReplicas).Should(BeEquivalentTo(1))
+	}, existingClusterTimeout, existingClusterInterval).Should(Succeed())
+
+	gvk := schema.GroupVersionKind{
+		Group:   "",
+		Version: "v1",
+		Kind:    "Pod",
+	}
+
+	restClient, err := apiutil.RESTClientForGVK(gvk, false, restConfig, serializer.NewCodecFactory(scheme.Scheme))
+	Expect(err).To(BeNil())
+
+	Eventually(func(g Gomega) {
+		execReq := restClient.
+			Post().
+			Namespace(namespace).
+			Resource("pods").
+			Name(podName).
+			SubResource("exec").
+			VersionedParams(&corev1.PodExecOptions{
+				Container: brokerCrName + "-container",
+				Command:   []string{"curl", "-H", "Origin:", "http://" + podName + ":8161", "-u", "anyuser:anypasswd", "http://" + podName + ":8161/console/jolokia/read/org.apache.activemq.artemis:broker=\"amq-broker\",address=\"" + addressName + "\",component=addresses,queue=\"" + queueName + "\",routing-type=\"" + routingType + "\",subcomponent=queues/" + attrName}, //curl -H "Origin: http://localhost:8161" -u guest:guest http://localhost:8161/console/jolokia/list
```

Comment:
maybe pull the comment over this Command or remove - it is way too long a line!

Reply:
An alternative  to avoid to run the command from the pod is using the jolokia client, i.e.
https://github.com/artemiscloud/activemq-artemis-operator/blob/main/controllers/activemqartemisaddress_controller_test.go#L533
```
						jolokia := jolokia.GetJolokia(pod.Status.PodIP, "8161", "/console/jolokia", "", "", "http")
						data, err := jolokia.Read("org.apache.activemq.artemis:broker=\"amq-broker\",address=\"" + addressName + "\",component=addresses,queue=\"" + queueName + "\",routing-type=\"" + routingType + "\",subcomponent=queues/" + attrName")

```

Reply:
yep didn't noticed the comment is added to the same line. 
I'll try to use the jolokia client as it's merged and convenient.